app/api/tools/route.ts

```diff
@@ -3,7 +3,11 @@ import prisma from "@/lib/prisma";
 import { Tool } from "@/types/schemaTypes";
 
 export async function GET(req: NextRequest) {
-  const tools:Tool[] = await prisma?.item.findMany({ take: 10 });
+  const tools:Tool[] = await prisma?.item.findMany({ 
```

Comment:
So line 6 after making suggested changes to the type file you can remove the type Tool[] because prisma will now create the return type for us.